prime-router/docs/getting_started.md

```diff
@@ -329,3 +332,41 @@ Alternatively, inject them in your terminal with (useful for using the CLI):
 ```bash
 export $(cat ./.vault/env/.env.local | xargs)
 ```
+
+
+## TroubleShooting
+
+### prime test Utility
+
+The prime-router comes packaged with a executable that can help in finding misconfigurations and other problems with the appliciation.
+
+Use the following command to launch the tool. 
+
+```shell
+./prime-router/prime test
```

Comment:
```suggestion
cd prime-router
./prime test
```

Sadly, it looks like our ./prime cli only works if you run in in the prime-router directory.

Reply:
Updated